### DISCUSSION TITLE
one flag to pause all deployments/pods?

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-cd#8468] [Q&A]
is there any setting I can change in yaml file that will cause all deployments/pods to stop (or not start)?

use case:
all my pods connect to remote services (ie databases).
on creation of new eks cluster, there won't be permissions for this new eks cluster to the remote services so the pods would fail but I would prefer to not generate all the failures. is there some one liner I can give in my app-of-apps yaml to say don't deploy anything or scale-to-0 or pause everything?

**Comment by RomanBats** [comment]
It is not the ArgoCD`s responsibility to control that. But one thing that you can do from the ArgoCD perspective is when you install App Of Apps you can specify [automatic sync](https://argo-cd.readthedocs.io/en/stable/user-guide/auto_sync/#automated-sync-policy) to be turned off. But in this case, ArgoCD will not install any resources and will wait until you press "sync" manually.

Another thing that I would rather do is to define a setting in the root Argo application, like "replicas: 0", and pass that setting to each child Argo application that would use that setting in each Deployment Kubernetes resource. And if you are ready to deploy all workloads you would set a positive number to the "replicas" setting, or just remove it to make a Deployment use default value suitable for this particular deployment.
